heka/meta/heka.yml

```diff
@@ -68,6 +68,16 @@ metric_collector:
       module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
       preserve_data: false
       message_matcher: "Type == 'heka.all-report'"
+    influxdb_accumulator:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/filters/influxdb_accumulator.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      preserve_data: false
+      message_matcher: "Fields[aggregator] == NIL && Type =~ /metric$/"
+      ticker_interval: 1
+      config:
+        tag_fields: "deployment_id environment_label hostname tenant_id user_id"
```

Comment:
Right. I removed `hostname` as it is currently in StackLight MOS. See https://github.com/openstack/fuel-plugin-lma-collector/blob/357fc8c52177a90d40a2bca7e86247fca93a1247/deployment_scripts/puppet/manifests/base.pp#L353.